doc/content/integrations/peering/_index.md

```diff
@@ -21,14 +21,22 @@ Peering is useful for all LoRaWAN public and private network operators. It can i
 
 Your network can be a forwarder and a home network. Forwarder networks have physical gateway infrastructure and home networks have end devices. Most LoRaWAN networks typically have gateways and end devices, so that they can be configured to play both roles. You can also have networks with only gateway infrastructure, configured as forwarder, or only with end devices, configured as home network.
 
-As a forwarder, your network offloads traffic that has been received by your gateways but that is not intended for your network. The offloading goes to a Packet Broker: a LoRaWAN traffic exchange or LoRaWAN roaming hub for {{% tts %}}.
+As a forwarder, your network offloads traffic that has been received by your gateways but that is not intended for your network. The offloading goes to a [Packet Broker]({{< ref "/integrations/peering#packet-broker" >}}) - a LoRaWAN traffic exchange or LoRaWAN roaming hub for {{% tts %}}.
 
 ### Packet Broker
 
-[Packet Broker](https://www.packetbroker.org) is a global backbone for LoRaWAN traffic. It is designed to exchange traffic securely between LoRaWAN networks. Packet Broker allows for individual packet selection; networks do not get charged for traffic they did not consume. Packet Broker separates traffic routing from billing and clearing; networks are free to put commercial agreements in place to settle balances. Packet Broker also separates payload from metadata; networks only get charged for the value they need. Finally, {{% tts %}} has native support for Packet Broker and can access the global coverage provided by The Things Network public community network.
+[Packet Broker](https://www.packetbroker.org) is a global backbone for LoRaWAN traffic. It is designed to securely exchange traffic between LoRaWAN networks. 
```

Comment:
I like this but we typically don't separate stuff in to bullets. If you want to keep it, remove the extra newlines at least.

Reply:
I'll keep it, I saw the bullets here too https://thethingsstack.io/integrations/aws-iot/default/ when highlighting the key features, so I guess that's fine